### PR TITLE
Mw/gh actions

### PR DESCRIPTION
**Pull request by michaelwood**
Thoughts on Integration tests here?

### CODE REVIEW

**Comment by Bjwebb** [comment]
I'm definitely up for trying this.

Not sure how we handle the case where you've deliberately made a breaking change, but have branches ready to go in each cove.

**Comment by odscjames** [comment]
Ooh, integration tests are interesting. Would the thinking be that we would set GitHub to not require Integration Tests to pass (only CI & Lint need to pass)? I'm not sure how it could work if we did - but it would still be really good to get advance notification of issues when doing the P.R. tho. 

Is there a reason lint only runs on 3.5 and 3.8?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # libcove doesn't support Windows.
+        # https://github.com/OpenDataServices/lib-cove/issues/53
+        os: [macos-latest, ubuntu-latest]
```
Comment (odscjames) [inline .github/workflows/ci.yml:10]: Do we need macos?

**Comment by odscjames** [comment]
Thought: Can we rename "ci" to "test"? It's just that "CI" is a general term that could cover many things. Having specific names like "integration", "lint" and "test" seem good.

**Comment by odscjames** [comment]
[Switch from Travis to GitHub Actions](https://trello.com/c/sLVHLfoo/46-switch-from-travis-to-github-actions)

**Comment by odscjames** [comment]
I've added a commit and tested. I'm putting this up for review, with the plan the Lint and Test stages will be required, and Integration won't be.

I haven't looked into why Integration is currently failing - but I think the C.I. is good, so I'd like to get the required ones in then we can look at integration (including adding 360 and IATI)

**Review by Lathrisk — APPROVED** [review]
This looks fine to me if we're currently ignoring integration tests.

**Pull request merged**